### ISSUE TITLE
Investigate data in report_test_coverage_by_project_prod

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/test-dashboard#27]
The graphs for report_test_coverage_by_project_prod view show that there are untriaged tests for Fenix. 
Andi raised the flag since running the report on Test Rail there are no untriaged tests for Full Functional or Smoke test suites.

This is the report in Test Rail:
[testrail-report-5779.pdf](https://github.com/mozilla-mobile/test-dashboard/files/9491065/testrail-report-5779.pdf)

This is the data from the Data Base:
https://docs.google.com/spreadsheets/d/15sWdruOkRIo9Anhcbk7dvSXkki24XMaZWXeNEMQ37DU/edit?usp=sharing

In yellow I have highlighted the same data we see on Test Rail, but there is more for the same day Smoke test and untriaged, highlighted in green, and those are being sum up, given the incorrect numbers.
We need to understand why those in green appear and why they are added.

**Comment by AndiAJ** [comment]
Hi @isabelrios ,

As discussed on Slack there should not be any "Untriaged" test in our Smoke and Sanity test suite.

- Both the test rail report and the numbers (highlighted in 🟠 ) retrieved from the data base are correct. ✅ 

![image](https://user-images.githubusercontent.com/51314259/188558062-9636a9d6-71b8-4207-850e-d77f12df2f41.png)
![image](https://user-images.githubusercontent.com/51314259/188558707-efcc8f6c-50e1-44ad-bb76-1fb5920f71dc.png)

I've made a quick check and I think I've found root cause because of which there are extra values (highlighted in 🟢 ) retrieved from the data base in regards with our smoke test suite.

- **Those numbers look like they are from our "Features" test suite.** [Test Rail report](https://testrail.stage.mozaws.net/index.php?/reports/view/5782) ❌ 

![image](https://user-images.githubusercontent.com/51314259/188560102-067be6db-245b-49e9-b026-0372fce654f8.png)
![image](https://user-images.githubusercontent.com/51314259/188560394-972ad744-2bd2-47b1-b00b-2e63aec7b5b8.png)

Please share your thoughts.

**Comment by isabelrios** [comment]
yeah, that makes sense to understand why the numbers are wrong in looker. 
In order to fix it, not sure if we can remove the smoke from those tests, or if those are actually part of the smoke test if they should be part of the full functional... not sure about the solution yet...
Please if you don't mind share your ideas as you are more familiar with the tests :)

**Comment by isabelrios** [comment]
As a possible solution to prevent this error from happening in the future, we could add the `testrail_test_suites_id` for each project, that's going to identify the Full Functional test suite which is the one we want specific data for now. Unless the test suite is removed and recreated, that ID is not going to change. If that happens, we should know as that's a big change.
Wdyt @AndiAJ @rpappalax ?
